content/SCALE/SCALETutorials/SystemSettings/Advanced/DeveloperMode.md

```diff
@@ -0,0 +1,28 @@
+---
+title: "Developer Mode (Unsupported)"
+description: "Provides information on the unsupported SCALE developer mode and how to enable it."
+weight: 80
+alias:
+tags:
+---
+
+{{< hint type="warning" title="Unsupported Feature" >}}
+Developer mode is for developers only.
+Users that enable this functionality will not receive support on any issues submitted to iXsystems.
+
+Only enable when you are comfortable with debugging and resolving all issues encountered on the system.
+Never enable on a system that has production storage and workloads.
+{{< /hint >}}
+
+TrueNAS is an Open Source Storage appliance, not a standard Linux operating system that allows customization of the Operating System environment.
+By default, the root/boot filesystem and tools such as `apt` are disabled to prevent accidental misconfigurations that render the system inoperable or put your stored data at risk.
```

Comment:
```suggestion
By default, the root/boot filesystem and tools such as `apt` are disabled to prevent accidental misconfiguration that renders the system inoperable or puts stored data at risk.
```